Makefile

```diff
@@ -213,6 +215,7 @@ else
 generate: tools-for-generate
 	@printf "\nFor more info on the generate command, Run 'make generate PRINT_HELP=y'\n"
 	@REPO_ROOT=$(REPO_ROOT) VGOPATH=$(VGOPATH) LOGCHECK_DIR=$(LOGCHECK_DIR) hack/generate.sh --what "$(WHAT)" --codegen-groups "$(CODEGEN_GROUPS)" --manifests-dirs "$(MANIFESTS_DIRS)" --mode "$(MODE)"
+	@cd pkg/apis; REPO_ROOT=$(REPO_ROOT) VGOPATH=$(VGOPATH) ../../hack/generate.sh --what "manifests" --manifests-dirs "./pkg/apis" --mode "$(MODE)"
```

Comment:
According to my testing `make test` no longer executes the unit tests under `./pkg/apis/`. You could verify this by just running `make test` and ensuring that there won't be a package that contains `github.com/gardener/gardener/pkg/apis`. Maybe the same is true for other make such as `make format`, `make check`.